### PR TITLE
Remove ability to lock to angles and add ability to lock to 'natural' orientation.

### PR DESCRIPTION
**Pull request by mounirlamouri**
Solves the following bug: https://www.w3.org/Bugs/Public/show_bug.cgi?id=25329

This push has two commits:
"Remove the ability to lock to angles." that removes the support for locking to 0, 90, 180 and 270.
"Add 'natural' orientation lock type." that adds support for locking to 'natural' which is equivalent to locking to angle 0.

Please ignore the following commit:
"Define what the term 'screen' means in the context of the specification."

### CODE REVIEW

**Comment by mounirlamouri** [comment]
@marcoscaceres can you take a look at this? :)

File: index.html [hunk 1]
```diff
@@ -464,9 +475,8 @@
               <i>orientations</i>.
               </li>
               <li>The <a>user agent</a> MUST change how the viewport is drawn
-              such as the <a>current screen orientation type</a> or the
-              <a>current screen orientation angle</a> will be equal to
-              <i>orientation</i>.
+              such as the <a>current screen orientation type</a> will be equal
```
Comment (marcoscaceres) [inline index.html]: - such that

Comment (marcoscaceres) [inline index.html]: Or "so that"

File: index.html [hunk 2]
```diff
@@ -476,12 +486,10 @@
...
           </li>
-          <li>If neither the <a>current screen orientation type</a> nor the <a>
-            current screen orientation angle</a> values are part of
-            <i>orientations</i>, the <a>user agent</a> MUST change how the
+          <li>If the <a>current screen orientation type</a> is not part of <i>
+            orientations</i>, the <a>user agent</a> MUST change how the
             viewport is drawn such as the <a>current screen orientation
-            type</a> or the <a>current screen orientation angle</a> will be
-            equal to one of <i>orientations</i>' values.
+            type</a> will be equal to one of <i>orientations</i>' values.
```
Comment (marcoscaceres) [inline index.html:487]: Is that closing `</a>` there a stray tag?

File: index.html (hunk 1 above)
Comment (mounirlamouri) [inline index.html]: Done

File: index.html (hunk 2 above)
Comment (mounirlamouri) [inline index.html:487]: No. It's closing the <a> opened for "current screen orientation type".

**Comment by mounirlamouri** [comment]
I assume that it is fine to merge. Please open another issue if you had any other comments.

**Pull request merged**